packages/frontend/scss/message/_message.scss

```diff
@@ -98,6 +98,29 @@ $info-text-max-width: 400px;
           cursor: pointer;
         }
       }
+      &.call {
+        > .text {
+          margin-inline-end: 24px;
+          white-space: nowrap;
+          font-weight: bold;
+          span:last-child {
+            margin-inline-end: 6px;
+          }
+          &::after {
+            content: '';
+            position: absolute;
+            top: 18px;
+            transform: translateY(-50%);
+            width: 24px;
```

Comment:
Some vars would be nice

Reply:
Hmm, IDK, but if you prefer I don't mind